src/halmos/sevm.py

```diff
@@ -2051,21 +2071,26 @@ def arith2(self, ex: Exec, op: int, w1: Word, w2: Word, w3: Word) -> Word:
     def mk_storagedata(self) -> StorageData:
         return self.storage_model.mk_storagedata()
 
-    def sload(self, ex: Exec, addr: Any, loc: Word) -> Word:
-        val = self.storage_model.load(ex, addr, loc)
-        ex.context.trace.append(StorageRead(addr, loc, val))
+    def sload(self, ex: Exec, addr: Any, loc: Word, transient: bool = False) -> Word:
+        storage = ex.storage if not transient else ex.transient_storage
```

Comment:
nit: avoid double negation

```suggestion
        storage = ex.transient_storage if transient else ex.storage
```

Reply:
https://github.com/a16z/halmos/pull/463/commits/2d64ce5b09c03a2b94e9308dc8abb0cee4ac393a